packages/extension-ui/src/Popup/Accounts/EditAccountMenu.tsx

```diff
@@ -3,41 +3,177 @@
 
 import type { ThemeProps } from '../../types';
 
-import React from 'react';
+import React, { useCallback, useContext, useState } from 'react';
+import CopyToClipboard from 'react-copy-to-clipboard';
+import { RouteComponentProps, withRouter } from 'react-router';
 import styled from 'styled-components';
 
+import { AccountJson } from '@polkadot/extension-base/background/types';
+import { EditMenuCard, Identicon } from '@polkadot/extension-ui/components';
+import useMetadata from '@polkadot/extension-ui/hooks/useMetadata';
+import { IconTheme } from '@polkadot/react-identicon/types';
+
+import forgetIcon from '../../assets/forget.svg';
+import subAccountIcon from '../../assets/subAccount.svg';
+import { Switch } from '../../components';
+import { AccountContext, SettingsContext } from '../../components/contexts';
+import useToast from '../../hooks/useToast';
 import useTranslation from '../../hooks/useTranslation';
+import { showAccount } from '../../messaging';
 import Header from '../../partials/Header';
+import { ellipsisName } from '../../util/ellipsisName';
 
-interface Props extends ThemeProps {
+interface Props extends RouteComponentProps<{ address: string; isExternal: string }>, ThemeProps {
   className?: string;
 }
 
-function EditAccountMenu({ className }: Props): React.ReactElement<Props> {
+function EditAccountMenu({
+  className,
+  match: {
+    params: { address, isExternal }
```

Comment:
`isExternal`, a condition, should not be a param but a query
https://stackoverflow.com/a/31261026